hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/volumes/volumes.tsx

```diff
@@ -410,11 +427,11 @@ export class Volumes extends React.Component<Record<string, object>, IVolumesSta
             loading={loading}
             pagination={paginationConfig}
             rowKey='volume'
-            scroll={{x: true, y: false, scrollToFirstRowOnChange: true}}
-            locale={{filterTitle: ""}}
+            scroll={{ x: 'max-content', scrollToFirstRowOnChange: true }}
```

Comment:
@devabhishekpal Just noticed that this change does not exist in the master. Any reason for this change?

Reply:
Similar changes in `buckets.tsx`, `datanodes.tsx`, `pipelines.tsx`

Reply:
Hi @ivandika3, this is merged but this change is good to have and I believe it is now present in 1.4 post this PR merge.
So the reason we change from `x: true` to `x: 'max-content'` is because with scroll with `x: true` means the table is scrollable along the x-axis i.e horizontally.
However in this setting the column widths are not properly set and we might face text wrap.

Setting it to max-content automatically adjusts the width to preferred size allowing columns to grow as required

Reply:
Current master does have this change present.
Sample ref: https://github.com/apache/ozone/blob/2401d27f68d6c6e776e106418a3302b71bb3297c/hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/buckets/buckets.tsx#L619

Reply:
@devabhishekpal Thanks for the explanation. I agree that the improvement is good to have.

> Current master does have this change present.

Yes, in `bucket.tsx` it exists, but `volumes.tsx` does not have this change. 

If the change is intended, it's nice to raise a patch to master so that the subsequent release based on master branch will have the change as well, to prevent regression between releases.

Reply:
Yup as a part of the v2 migration we are having this change present.
So the future plan is once all the pages have completed migration we will remove old ui and completely use the new UI.
As a part of this we have few fixes as well as optimisations in the V2 UI as well.